lib/ui/src/modules/ui/components/stories_panel/sub_header.js

```diff
@@ -0,0 +1,36 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { baseFonts } from '@storybook/components';
+
+const wrapperStyle = {
+  marginTop: 20,
+};
+
+const headingStyle = {
+  ...baseFonts,
+  textTransform: 'uppercase',
+  // letterSpacing: '1.5px',
+  fontSize: '12px',
+  fontWeight: 'normal',
+  color: '#828282',
+  textAlign: 'left',
+  padding: '5px 13px',
+  margin: 0,
+  overflow: 'hidden',
+};
+
+const SubHeader = ({ name }) => (
+  <div style={wrapperStyle}>
+    <h4 style={headingStyle}>{name}</h4>
+  </div>
+);
+
+SubHeader.defaultProps = {
+  name: '',
```

Comment:
what do you think about changing `name` to `children` ? I think it would be nicer for use. And I think it should be `required`.

Reply:
Agreed! Initially copied stories_panel/header.js props interface. But I prefer children prop in such situations. So changed it.